src/app/shared/components/header/header.html

```diff
@@ -0,0 +1,68 @@
+<mat-toolbar
+  aria-label="Main navigation"
+  color="primary"
+  role="banner">
+  <div class="header-container">
+    <a
+      aria-label="FoxFlow Home"
+      class="logo"
+      routerLink="/">
+      <img
+        alt="FoxFlow Logo"
+        height="48"
+        ngSrc="assets/images/logo.png"
+        width="48" />
+    </a>
+    <span class="spacer"></span>
+    @if (!isAuthenticated && (isMobile$ | async) === false) {
+    <div class="nav-links">
+      <a
+        aria-label="Log in"
+        mat-button
+        routerLink="/login"
+        routerLinkActive="active">
+        {{ 'AUTH.LOGIN' | translate }}
+      </a>
+      <a
+        aria-label="Register"
+        mat-button
+        routerLink="/register"
+        routerLinkActive="active">
+        {{ 'AUTH.REGISTER' | translate }}
+      </a>
```

Comment:
Add primary button style.

```suggestion
      <a
        aria-label="Register button"
        matButton="filled"
        routerLink="/register"
        routerLinkActive="active">
        {{ 'AUTH.REGISTER' | translate }}
      </a>
```

Reply:
Thank you for pointing this out! Fixes are applied.